pkg/kubelet/volume/cache/desired_state_of_world.go

```diff
@@ -203,6 +217,25 @@ func (dsw *desiredStateOfWorld) AddPodToVolume(
 	return volumeName, nil
 }
 
+func (dsw *desiredStateOfWorld) MarkVolumesReportedInUse(
+	reportedVolumes []api.UniqueVolumeName) {
+	dsw.Lock()
+	defer dsw.Unlock()
+
+	reportedVolumesMap := make(
+		map[api.UniqueVolumeName]bool, len(reportedVolumes) /* capacity */)
+
+	for _, reportedVolume := range reportedVolumes {
+		reportedVolumesMap[reportedVolume] = true
+	}
+
+	for volumeName, volumeObj := range dsw.volumesToMount {
+		_, volumeReported := reportedVolumesMap[volumeName]
+		volumeObj.reportedInUse = volumeReported
+		dsw.volumesToMount[volumeName] = volumeObj
```

Comment:
the comment for this method indicates that we're resetting it to false if it's not in the list of reportedVolumes. I guess we start off setting it to false, can you clarify by modifying the comment?